src/api/mail/controller.js

```diff
@@ -0,0 +1,39 @@
+import { sendJson } from 'utils/api';
+import config from 'config';
+import HttpError from 'node-http-error';
+import HttpStatus from 'http-status-codes';
+
+const Mailchimp = require('mailchimp-api-v3');
+const md5 = require('md5');
+
+export const getEmailAddressHash = emailAddress => md5(emailAddress.trim().toLowerCase());
+
+export const updateEmailAddressStatus = (req, res, next) => {
```

Comment:
it should be possible to eliminate `try-catch` structure and implement the method using `then`-chaining only